reporter_test.go

```diff
@@ -28,7 +28,7 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"github.com/uber/jaeger-lib/metrics"
-	"github.com/uber/jaeger-lib/metrics/metricstest"
+	mTestutils "github.com/uber/jaeger-lib/metrics/metricstest"
```

Comment:
why override the `metricstest` name?

Reply:
Not sure - so have changed - may have been a conflict previously.